org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/panel/InfoPanel.java

```diff
@@ -2637,7 +2640,7 @@ public void dispose(boolean ok)
     {
     	if (log.isLoggable(Level.CONFIG)) log.config("OK=" + ok);
         m_ok = ok;
-
+        saveWlistBoxColumnWidth(this.getFirstChild());
```

Comment:
here we should check that wlistboxname is not empty/null

Reply:
This is done in the function saveColumnWidth in the class WListbox, so nothing is save without a valid wlistboxname.